tests/php/ORM/DataListTest.php

```diff
@@ -1220,6 +1222,107 @@ public function testFilterByNullCase()
         $this->assertSQLNotContains('"DataObjectTest_Fan"."Email" IS NOT NULL', $items9->sql());
     }
 
+    public function testAggregateDBName()
+    {
+    	$filter = new ExactMatchFilter(
+    		'Comments.Count()'
+    	);
+    	$filter->setModel(new DataObjectTest\Team());
+    	$this->assertEquals('COUNT("DataObjectTest_Team"."ID")', $filter->getDBName());
+
+    	foreach(['Comments.Max(ID)', 'Comments.Max( ID )', 'Comments.Max(  ID)'] as $name) {
+	    	$filter = new ExactMatchFilter($name);
+	    	$filter->setModel(new DataObjectTest\Team());
+	    	$this->assertEquals('MAX("DataObjectTest_Team"."ID")', $filter->getDBName());
+
+    	}
+    }
+
+    public function testAggregateFilterExceptions()
+    {
+		$ex = null;
+		try {
+			$filter = new ExactMatchFilter('Comments.Max( This will not parse! )');
+		} catch(\Exception $e) {
+			$ex = $e;
```

Comment:
It looks like this assignment is unnecessary as the resulting var is never used.